aggregate_spatial.json

```diff
@@ -49,7 +49,13 @@
                         "optional": true,
                         "default": null
                     }
-                ]
+                ],
+                "returns": {
+                    "description": "Return the value that should be stored in the data cube.",
```

Comment:
> Return the value that should be stored in the data cube.

The "Return" verb at the beginning seems a bit out of place to me, I think the description works without it.

And maybe for clarity: "stored in the output data cube" (or "resulting data cube")

Reply:
Thanks, indeed this can be written a bit more compact. Will be fixed in the next commit.

It seems we have not used anything like "output ..." or "resulting ..." yet in the processes. In the "returns" of the process, it also just says "A vector data cube". So there's not really a consistent phrase we can "refer" to. In array_apply it says "An array with the newly computed values.", which also doesn't make a short referable phrase. On the other hand, I'm not even sure the clarification is really needed.

Reply:
yeah, maybe what triggered me is the verb "store" as well. "Store in the data cube" somewhat sounds as in-place replacement to me for some reason. While more generic, "use" is less suggestive maybe?

Reply:
Indeed, store sounds like an in-place replacement.
I'm not exactly sure why, but "The value to be used in the data cube/array" sounds strange. "use" seems to generic and it could be unclear what that actually means. Would "The value to be stored in a new data cube/array" work? That would be somewhat aligned with the process return value description ("A data cube with the newly computed values.") in most cases.

Reply:
Is commit https://github.com/Open-EO/openeo-processes/pull/215/commits/ec5f84585bdf8c61df66806d5e8548c04aa6509b fine for you? I'm pretty happy with the improvement.

Reply:
yes that's already better

another issue with "store" is that it also sound like "persist or "store to disk" (like `save_result`), which is not what you to suggest I guess. 

I had a quick search on thesaurus.com and kind of prefer these alternatives: "gather", "assemble", "collect"

Reply:
I think store (here: in memory) is fine for me and it should be clear from the context, but a simple alternative would be "set", I think.

Reply:
@soxofaan I've changed most occurances of 'store' to set, used or added, see https://github.com/Open-EO/openeo-processes/pull/215/commits/8be67ff27b1b29933d4df683ffec97735ade052e